app/src/main/java/org/mozilla/fenix/home/HomeFragmentStore.kt

```diff
@@ -17,9 +18,10 @@ import org.mozilla.fenix.components.tips.Tip
  * The [Store] for holding the [HomeFragmentState] and applying [HomeFragmentAction]s.
  */
 class HomeFragmentStore(
-    initialState: HomeFragmentState = HomeFragmentState()
+    initialState: HomeFragmentState = HomeFragmentState(),
+    middleware: List<Middleware<HomeFragmentState, HomeFragmentAction>> = emptyList()
```

Comment:
I think you want to call this `middlewares`. You also done this for the TabsTrayStore.